lib/src/test/java/io/ably/lib/test/realtime/RealtimeConnectFailTest.java

```diff
@@ -166,23 +167,32 @@ public void connect_while_disconnected() {
 	}
 
 	/**
-	 * Verify that the connection enters the disconnected state, after a token
-	 * used for successful connection expires
+	 * Verify that the server issues reauth message 30 seconds before token expiration time, authCallback is
+	 * called to obtain new token and in-place re-authorization takes place with connection staying in connected
+	 * state
 	 */
 	@Test
-	public void connect_token_expire_disconnected() {
+	public void connect_token_expire_inplace_reauth() {
```

Comment:
Behaviour has changes and so did the test. I can add a new test but it will have to wait 30+ seconds for the disconnect message and I can't fix it by changing default timeouts on the client side.